tests/resources/test_issue.py

```diff
@@ -26,9 +26,12 @@ def test_issue(self):
         self.assertEqual(issue.key, self.issue_1)
         self.assertEqual(issue.fields.summary, "issue 1 from %s" % self.project_b)
 
+        self.assertEqual(issue.get_field('summary'), "issue 1 from %s" % self.project_b)
```

Comment:
Please do not do this. You should not edit all the tests to include this new access method. This does not actually 'test' your code's edge cases.

Create a new function that only tests this `get_field()` function, where you create a new issue and test some example fields, you can test a string field (e.g. summary), a list field (e.g. labels) and the project field and ensure that the type is a `Project`.
This means you are able to verify that you can still get the correct types from various different fields. Including the custom types such as `Project`.


Then you should also create at least one (ideally two) separate function that tests incorrect fields, e.g. a field that start with an underscore - to ensure your attribute error is raised and a field that definitely won't exist.

Tests are not just a box ticking exercise !

Reply:
Thanks for the feedback, I think I have underestimated the effort there have been put into the unit test already made, I will put some more effort into it tomorrow or the day after tomorrow (hopefully)

Reply:
No problem ! Feel free to ask me any further questions if anything is unclear and feel free to take your time, we all have stuff going on (and you should enjoy yourself too !)